test/integration_tests/test_load_executor.py

```diff
@@ -47,5 +47,5 @@ def test_should_load_video_in_table(self):
         # Do we have select command now?
         metadata = CatalogManager().get_dataset_metadata("", "MyVideo")
         actual_batch = list(StorageEngine.read(metadata))[0]
-        expected_batch = list(create_dummy_batches())
+        expected_batch = list(create_dummy_batches())[0]
```

Comment:
Why are picking index 0?

Reply:
Because it returns a list of batch. You can either remove or keep [0] index for both actual_batch and expected_batch.

Reply:
Actual_batch is also a list and so is expected_batch
We can compare lists

Reply:
yeah, right. But i think in the current master, acutal_batch is "Batch" and expected_batch is list, so this testcase will fail.